### PR TITLE
Config livemode

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**What change(s) does this PR introduce?** 

Allows users to potentially include testing/non-live customer data (for testing purposes)

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide explanation how the change is non breaking below.)

**Is this PR in response to a previously created Issue**

- [x] Yes, Issue https://github.com/fivetran/dbt_stripe_source/issues/19
- [ ] No 

**How did you test the PR changes?** 

 
- [x] CircleCi  
- [x] Other (please provide additional testing details below) directly in bigquery

**Select which warehouse(s) were used to test the PR**
 
- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🚚 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
Your solution works great, but I think we need to gain more context from the customer in order to move forward with this PR. The customer responded to their Issue #19 saying they want to see their test customers next to live customers. I am suspicious to think that a test customer will not be tied to `livemode` invoices, payments, charges, etc. If a test customer is also tied to other test objects then we will need to add this functionality to all other tables that allow the `livemode` feature. 

Our next steps for this enhancement would be to go back to the customer and ask if this solution is sufficient for their request, or if we will want to apply a blanket `exclude_test` variable that allows users to enable or disable the livemode functionality across the entire package. We will then want to determine if the customer is intending for any of these changes to be replicated in downstream models. I just did a quick check and notice we have no `is_livemode` fields so it would be impossible to determine in aggregates which records are test and which are not within the transform package.

File: README.md [hunk 1]
```diff
@@ -71,7 +73,18 @@ models:
 
 *Read more about using custom schemas in dbt [here](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas).*
 
-### Contributions ###
+### Running on Live vs Test Customers
+By default, this package will run on non-test data (`where livemode = true`) from the source Stripe tables. However, you may want to include and focus on test data when testing out the package or developing your analyses. To run on _only_ test data, add the following configuration to your `dbt_project.yml` file:
```
Comment (fivetran-joemarkiewicz) [inline README.md:77]: Did you intend for `_only_` to have the underscores?

File: models/tmp/stg_stripe__invoice_line_item_tmp.sql [hunk 2]
```diff
@@ -2,3 +2,5 @@
 
 select * 
 from {{ var('invoice_line_item') }}
+
+{{ livemode_predicate() }}
```
Comment (fivetran-joemarkiewicz) [inline models/tmp/stg_stripe__invoice_line_item_tmp.sql:6]: It looks like our ERD shows `livemode` as `live_mode`. Let's confirm with the PM what the is accurate here and if we should make an adjustment to the ERD.

image

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Looks good! I just had one comment we should look into before release regarding the naming of `livemode` within `invoice_line_item`

File: README.md (hunk 1 above)
Comment (fivetran-jamie) [reply README.md:77]: i did at the time but maybe i should just remove...

Comment (fivetran-joemarkiewicz) [reply README.md:77]: Yeah I like it italicized, but let's remove the underscores.

File: models/tmp/stg_stripe__invoice_line_item_tmp.sql (hunk 2 above)
Comment (fivetran-jamie) [reply models/tmp/stg_stripe__invoice_line_item_tmp.sql:6]: yeah pretty sure the ERD is incorrect -- we've been using `livemode` in the columns [macro](https://github.com/fivetran/dbt_stripe_source/blob/master/macros/get_invoice_line_item_columns.sql#L11)

Comment (fivetran-joemarkiewicz) [reply models/tmp/stg_stripe__invoice_line_item_tmp.sql:6]: Good call! Looks good then!

**Pull request merged**